web/themes/joinup/prototype/sass/components/_form-table.sass

```diff
@@ -9,7 +9,7 @@
   margin-left: 25px
 
 .form-table__cell
-  padding: 0 15px
+  padding: 15px
```

Comment:
It's always better to be specific, than loose, it'd recommend leaving this specific.